### PR TITLE
New Exchanger added: Bitreserve.org

### PR DESCRIPTION
**Pull request by joseccnet**
New Exchanger added: Bitreserve.org
Contribution from @joseccnet

Checked working with DataModuleTester.

Please add for the main Android App.

I´m just a user from bitreserve.org

Good project @mobnetic !!

### CODE REVIEW

**Comment by joseccnet** [comment]
Adding the Argentine Peso (ARS), Brazilian Real (BRL), United Arab Emirates Dirham (AED) and Philippine Peso (PHP) to bitreserve.org

Checked working with DataModuleTester.

Please add for the main Android App.

File: DataModule/src/com/mobnetic/coinguardian/model/market/OkcoinFuturesPrice.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import com.mobnetic.coinguardian.model.CheckerInfo;
+import com.mobnetic.coinguardian.model.Market;
+import com.mobnetic.coinguardian.model.Ticker;
+import com.mobnetic.coinguardian.model.currency.Currency;
+import com.mobnetic.coinguardian.model.currency.VirtualCurrency;
+
+public class OkcoinFuturesPrice extends Market {
+
+	private final static String NAME = "OKCoin Futures Price";
+	private final static String TTS_NAME = "Ok coin feature price";
```
Comment (btcdrak) [inline DataModule/src/com/mobnetic/coinguardian/model/market/OkcoinFuturesPrice.java]: "OKCoin futures price"

File: DataModule/src/com/mobnetic/coinguardian/model/currency/VirtualCurrency.java [hunk 2]
```diff
@@ -36,6 +36,9 @@
 	public static final String BQC = "BQC";
 	public static final String BTB = "BTB";
 	public static final String BTC = "BTC";
+	public static final String BTC_1WeekFutures = "BTC This Week Futures";
```
Comment (btcdrak) [inline DataModule/src/com/mobnetic/coinguardian/model/currency/VirtualCurrency.java]: Really needs to be BTCFUT1W  = "BTCFUT1W" 
etc.

File: DataModule/src/com/mobnetic/coinguardian/model/currency/VirtualCurrency.java [hunk 3]
```diff
@@ -170,6 +173,9 @@
 	public static final String LOT = "LOT";
 	public static final String LTB = "LTB";
 	public static final String LTC = "LTC";
+	public static final String LTC_1WeekFutures = "LTC This Week Futures";
```
Comment (btcdrak) [inline DataModule/src/com/mobnetic/coinguardian/model/currency/VirtualCurrency.java]: Really needs to be LTCFUT1W  = "LTCFUT1W" 
etc.

**Comment by mhiller** [comment]
three things -
1.) I know okcoin api calls them this_week, next_week, quarter but shouldnt okcoin be called weekly ,biweekly and quaterly futures to match the site ?
2.) the okcoin check was attached bitreserve check in by joseccnet . is this ok with mobnetic ? some project demand a seperate checkin for log/merge/fast-forward purposes
3.) +1 both commits baring no issue to the above.

**Comment by joseccnet** [comment]
Just few minutes ago, I start to studing about git(branches, and rebase).
In progress, sorry. Im newbie in this part of git.

About 1) which part of the code to modify "weekly ,biweekly and quaterly"?
its not clear for me. I can change if you tell me. The thing to considere,
is to make it easy to understand to the final user.

About 2. I think, im going to create branches, as you recommend.

please give me chance to read some of git, I will try to do as you told me.

Reading...


On Sun, Sep 20, 2015 at 9:29 PM, mhiller <notifications@github.com> wrote:

> three things -
> 1.) I know okcoin api calls them this_week, next_week, quarter but
> shouldnt okcoin be called weekly ,biweekly and quaterly futures to match
> the site ?
> 2.) the okcoin check was attached bitreserve check in by joseccnet . is
> this ok with mobnetic ? some project demand a seperate checkin for
> log/merge/fast-forward purposes
> 3.) +1 both commits baring no issue to the above.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mobnetic/BitcoinChecker/pull/172#issuecomment-141861909>
> .
>



-- 
Ing. José  C.

**Comment by btcdrak** [comment]
continued in #174

**Pull request closed without merging**